doc/sphinx/changes.rst

```diff
@@ -14,6 +14,91 @@ Version 8.12
 Summary of changes
 ~~~~~~~~~~~~~~~~~~
 
+|Coq| version 8.12 integrates many quality-of-life improvements,
+in particular with respect to notations, scopes and implicit arguments,
+along with many bug-fixes and a reworked reference manual. The main changes
+include:
+
+- New `binder notation`__ for non-maximal implicit arguments using :g:`[ ]`
+  allowing to set and see the implicit status of arguments immediately.
+- New notation :g:`Inductive I A | x : s := ...` to distinguish the
+  `uniform`__ from the non-uniform parameters in inductive definitions.
+- More robust and expressive treatment of `implicit inductive`__
+  parameters in inductive declarations.
+- Improvements in the treatment of implicit arguments and partially applied
+  constants in `notations`__, parsing of hexadecimal number notation and better
+  handling of scopes and coercions for printing.
+- A correct and efficient `coercion coherence`__ checking algorithm, avoiding
+  spurious or duplicate warnings.
+- An improved :g:`Search` `command`__ which accepts complex queries. Note that
+  this takes precedence over the now deprecated `ssreflect search`__.
+- Many additions and improvements of the `standard library`__.
+- Overhaul of the `reference manual`__ giving a more coherent view of the system,
+  along with consistent grammar notations that are up-to-date with Coq's grammar.
```

Comment:
```suggestion
- Improvements to the `reference manual`__ include a more logical organization of chapters
  along with updated syntax descriptions that match Coq's grammar in most
  but not all chapters.
```

"overhaul" incorrectly suggests we're done.

My best guess at the moment: Ltac chapter will be done within the next few days.  We may also be able to update Ltac2 and/or the tactics chapter in time for 8.12.  That leaves only SSR and several less-central chapters that don't have a lot of syntax to update.

Reply:
Ltac2 might be feasible for 8.12.0, but for the tactics chapter I think it is best to target 8.13.

Reply:
Ok. Let's not oversell it, I switch to your wording @jfehrle